api/app/models/annotation.rb

```diff
@@ -31,6 +31,10 @@ class Annotation < ApplicationRecord
   scope :by_formats, lambda { |formats|
     where(format: formats) if formats.present?
   }
+  scope :excluding_private, lambda { |creator|
```

Comment:
I added a FIXUP commit that removes this, as it does not appear to be used anywhere, and seems logically incomplete.